stbt/keyboard.py

```diff
@@ -327,3 +337,24 @@ def _add_weights(G):
                 # take a shortcut through here.
                 for target in targets:
                     G[node][target]["weight"] = 100
+
+
+def _join_with_commas(items, last_one=", "):
```

Comment:
Presumably there will be a `last_one=" and "` in a future commit...

No, I didn't see it.  I don't see why this couldn't default to `" or "` or even have `" or "` hard-coded.

Reply:
It seemed clearer from the call-site.